cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -1191,12 +1208,18 @@ public MapAndStatus(T loadMap, K status) {
     private KafkaRebalanceAnnotation rebalanceAnnotation(Reconciliation reconciliation, KafkaRebalance kafkaRebalance) {
         String rebalanceAnnotationValue = rawRebalanceAnnotation(kafkaRebalance);
         KafkaRebalanceAnnotation rebalanceAnnotation;
+
         try {
-            rebalanceAnnotation = rebalanceAnnotationValue == null ?
-                    KafkaRebalanceAnnotation.none : KafkaRebalanceAnnotation.valueOf(rebalanceAnnotationValue);
+            if (rebalanceAnnotationValue == null) {
+                rebalanceAnnotation = KafkaRebalanceAnnotation.none;
+            } else if (rebalanceAnnotationValue.equals("none")) {
```

Comment:
So you do have logic for dealing with `none`. Why did you add that option to the enum?